server/database/build.js

```diff
@@ -1,12 +1,19 @@
-const { readFileSync } = require('fs');
-const path = require('path');
-const connect = require('./connection.js');
+const { readFile } = require('fs');
+const { join } = require('path');
+const { query } = require('./connection.js');
```

Comment:
query is a vague name that doesn't properly describe its purpose.

```dbBuildQuery``` is a longer name but is more clear.